pyiceberg/catalog/hive.py

```diff
@@ -121,6 +121,9 @@
 HIVE2_COMPATIBLE = "hive.hive2-compatible"
 HIVE2_COMPATIBLE_DEFAULT = False
 
+HIVE_KERBEROS_AUTH = "hive.kerberos-authorization"
```

Comment:
nit: `authentication` is different than `authorization` 
```suggestion
HIVE_KERBEROS_AUTH = "hive.kerberos-authentication"
```

Kerberos is authentication https://docs.cloudera.com/cdw-runtime/1.5.1/securing-hive/topics/hive_remote_data_access.html